WalletWasabi.Daemon/Rpc/WasabiJsonRpcService.cs

```diff
@@ -216,20 +216,31 @@ public object GetStatus()
 	}
 
 	[JsonRpcMethod("build")]
-	public string BuildTransaction(PaymentInfo[] payments, OutPoint[] coins, int feeTarget, string? password = null)
+	public string BuildTransaction(PaymentInfo[] payments, OutPoint[] coins, int? feeTarget = null, int? feeRate = null, string? password = null)
```

Comment:
Maybe we should be consistent with the app and accept `feeRate` up to the third decimal?

Reply:
It can be. In this PR the feeRate is satoshis per kilobyte. Should we change it to represent satoshis per byte too?

Reply:
The `feeRate` is in kb internally, but I don't see any problem. The caller has to provide the fee rate in sat per byte, so I believe the rest of your PR is perfect.
But if someone uses a custom fee estimator he might want to use a decimal fee rate.